e2e-tests/scheduled-backup/conf/cr.yaml

```diff
@@ -0,0 +1,138 @@
+apiVersion: pg.percona.com/v1
+kind: PerconaPGCluster
+metadata:
+  annotations:
+    current-primary: cluster1
+  labels:
+    crunchy-pgha-scope: cluster1
+    deployment-name: cluster1
+    name: cluster1
+    pg-cluster: cluster1
+    pgo-version: 0.2.0
+    pgouser: admin
+  name: cluster1
+spec:
+  database: pgdb
+  namespace: pgo
+  port: "5432"
+  user: pguser
+  disableAutofail: false
+  tlsOnly: false
+  standby: false
+  pause: false
+  userLabels:
+    pgo-version: "0.2.0"
+  pgPrimary:
+    image: perconalab/percona-postgresql-operator:main-ppg13-postgres-ha
+    resources:
+      requests:
+        memory: "128Mi"
+    tolerations: []
+    volumeSpec:
+      size: 1G
+      accessmode: ReadWriteOnce
+      storagetype: dynamic
+      storageclass: ""
+#      matchLabels: ""
+    expose:
+      serviceType: ClusterIP
+#      loadBalancerSourceRanges:
+#      annotations:
+#        pg-cluster-annot: cluster1
+#      labels:
+#        pg-cluster-label: cluster1
+  pmm:
+    enabled: false
+    image: percona/pmm-client:2.15.1
+    serverHost: monitoring-service
+    serverUser: admin
+    pmmSecret: cluster1-pmm-secret
+    resources:
+      requests:
+        memory: 200M
+        cpu: 500m
+#      limits:
+#        cpu: "1"
+#        memory: "400M"
+  backup:
+    image: perconalab/percona-postgresql-operator:main-ppg13-pgbackrest
+    backrestRepoImage: perconalab/percona-postgresql-operator:main-ppg13-pgbackrest-repo
+    resources:
+      requests:
+        memory: "48Mi"
+#      limits:
+#        cpu: "1"
+#        memory: "64Mi"
+    volumeSpec:
+      size: 1G
+      accessmode: ReadWriteOnce
+      storagetype: dynamic
+      storageclass: ""
+#      matchLabels: ""
+#    storages:
+#      my-gcs:
+#        type: gcs
+#        bucket: some-gcs-bucket
+#    storageTypes: [
+#      "gcs"
+#    ]
+#    repoPath: ""
+    schedule:
+      - name: "sch-backup"
+        schedule: "*/5 * * * *"
```

Comment:
make backup schedule more intensive for some period of time. We need to understand how fast and robust is the approach available (more like pxc)

Reply:
as we discussed, we don't need it here